wdlconfig.qmd

```diff
@@ -0,0 +1,58 @@
+
+# WDL Configuration Guide {{< iconify file-icons wdl >}} {#sec-wdlconfig}
+
+So as not to reinvent the wheel, WILDS WDL's should follow guidelines similar to the ones provided by [BioWDL](https://biowdl.github.io/styleGuidelines.html) and [WARP](https://broadinstitute.github.io/warp/docs/Best_practices/suggested_formats). However, because of the pedagogical "proof-of-concept" nature of the WILDS, they will not be identical and even differ significantly in a few places.
+
+## WILDS Philosophy
+
+- The mindset behind WILDS is for each repository to be a self-contained demonstration of a particular bioinformatic functionality:
+    1. Researcher scans the workflow to deem whether it is relevant to their needs.
+    2. Researcher clones the repository as is, makes minimal updates to the inputs, and easily executes the code locally or otherwise.
+    3. Researcher forks the repository and customizes it as necessary to fit their exact research needs.
+- To that end, WILDS WDL repositories will usually consist of a single WDL script containing the workflow as well as the tasks that make up the workflow.
+    - This contradicts the recommendations from BioWDL, i.e. tasks should be written in a separate script and imported into the workflow script as a module.
```

Comment:
My take is not to keep referring back to the BioWDL or WARP recommendations. But just to be very straight about what our rules are and this is what you should expect.